modules/cockroach/src/main/java/org/testcontainers/containers/CockroachContainer.java

```diff
@@ -0,0 +1,86 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.HttpWaitStrategy;
+
+import java.time.Duration;
+
+public class CockroachContainer<SELF extends CockroachContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
+    public static final String NAME = "cockroach";
+    public static final String IMAGE = "cockroachdb/cockroach";
+    public static final int REST_API_PORT = 8080;
+    public static final int DB_PORT = 26257;
+
+    private String databaseName = "test";
+    private String username = "root";
+    private String password = "";
```

Comment:
shouldn't there be withPassword() method?

Reply:
Interestingly, there is not a straightforward way to launch Cockroach with a custom password. As-is, this PR has it running in passwordless `insecure` mode.

As for other test container types we're not using secure passwords I think this is OK, but worth considering.

I'm tempted to add a `withPassword` method that just throws and `UnsupportedOperationException` for symmetry/visibility. 

WDYT?

Reply:
> Interestingly, there is not a straightforward way to launch Cockroach with a custom password. As-is, this PR has it running in passwordless insecure mode.

Yes, that is right.

> I'm tempted to add a withPassword method that just throws and UnsupportedOperationException for symmetry/visibility.

Sounds good to me.